# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/gtest/audioconduit_unittests.cpp
# Commit: e0a2e54e4de8
# Full Hash: e0a2e54e4de80813829cbcf9e4a5e4ff1285d451
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 9
# Description:
#   Bug 1654112 - Move WebRtcCallWrapper to dedicated files. r=ng
#   
#   This patch also renames it WebrtcCallWrapper, to better match neighboring
#   classes starting on "Webrtc".
#   
# ==============================================================================

diff -r 1965757ee924 -r e0a2e54e4de8 media/webrtc/signaling/gtest/audioconduit_unittests.cpp
--- a/media/webrtc/signaling/gtest/audioconduit_unittests.cpp	Thu Sep 02 13:22:03 2021 -0500
+++ b/media/webrtc/signaling/gtest/audioconduit_unittests.cpp	Thu Apr 08 11:31:43 2021 +0200
@@ -8,6 +8,7 @@
 #include "gtest/gtest.h"
 
 #include "AudioConduit.h"
+#include "WebrtcCallWrapper.h"
 
 #include "MockCall.h"
 
@@ -19,7 +20,7 @@
  public:
   AudioConduitTest()
       : mCall(new MockCall()),
-        mCallWrapper(WebRtcCallWrapper::Create(UniquePtr<MockCall>(mCall))),
+        mCallWrapper(WebrtcCallWrapper::Create(UniquePtr<MockCall>(mCall))),
         mAudioConduit(MakeRefPtr<WebrtcAudioConduit>(
             mCallWrapper, GetCurrentSerialEventTarget())) {}
 
@@ -29,7 +30,7 @@
   }
 
   MockCall* const mCall;
-  const RefPtr<WebRtcCallWrapper> mCallWrapper;
+  const RefPtr<WebrtcCallWrapper> mCallWrapper;
   const RefPtr<WebrtcAudioConduit> mAudioConduit;
 };
 